lib/commands/general.js

```diff
@@ -148,6 +148,32 @@ commands.getStrings = async function (language) {
   return this.apkStrings[language];
 };
 
+commands.startActivity = async function (appPackage, appActivity) {
+  return this.adb.startApp({
```

Comment:
what does this return? if nothing, we don't need it.

also, we need an `await` here